webapp/src/cljs/lipas/ui/map/utils.cljs

```diff
@@ -419,6 +419,9 @@
 (defn update-center!
   [{:keys [^js view] :as map-ctx}
    {:keys [lon lat] :as center}]
+  ;; TODO: By telling OL if the left side panel is open,
+  ;; it can correctly take this into account when focusing on coordinates.
+  ;; (set! (.-padding view) #js [0 0 0 530])
```

Comment:
Good point and please do this as well if there's time left after the MUI upgrade! There are a few different cases for the drawer width. Current logic is in `:lipas.ui.map.subs/drawer-width` subscription.